### ISSUE TITLE
[Feature request] Keyboard shortcut for TST in sidebar.

### ISSUE THREAD

**Issue by vonHabsi** [piroor/treestyletab#2431]
A number of the addons which use the sidebar have keyboard shortcuts which toggle the display of the sidebar, eg Ctrl-B for Bookmarks , Ctrl-E for Sidebery, Ctrl-Shift-9 for All Tabs Helper.

I had some user.js which implemented it but the changes in Firefox have disabled it.

Can something like that be implemented for TST?

Will it be easy to do something like that myself?

**Comment by irvinm** [comment]
If you go to TST options under "Keyboard Shortcuts" I believe the first item is what you are looking for.

**Comment by vonHabsi** [comment]
> If you go to TST options under "Keyboard Shortcuts" I believe the first item is what you are looking for.

Thanks. I think I configured it in the past and forgot about it, but it doesn't suit my need exactly.

The user.js I created used CSS to simply hide the sidebar so hiding and revealing was instantaneous, but the TST keyboard shortcut refreshes the treeview anew when it displays so it takes a few seconds before the tree is restored, and can be slow with a large number of tabs.

For TST it works as designed so I am closing the issue.
